### DISCUSSION TITLE
Task :react-native-gesture-handler:compileDebugKotlin FAILED

### DISCUSSION THREAD

**Discussion by HuzaiFa3030** [software-mansion/react-native-gesture-handler#2546] [Q&A]
Task :react-native-gesture-handler:compileDebugKotlin FAILED

"react-native": "0.70.3",
"react-native-gesture-handler": "^2.19.0",
"react-native-reanimated": "^3.3.0",

dependencies {
classpath("com.android.tools.build:gradle:7.2.1")
.....
}

**Comment by SavanExactink** [comment]
same issue, is there any fix for it?

**Reply by HuzaiFa3030** [reply]
Finally, I resolve this issue
just need to update your ### android/build.gradle

ext {
        buildToolsVersion = '31.0.0'
        minSdkVersion = 21
        compileSdkVersion = 33      
        targetSdkVersion = 33
....
}

its works for me. 
Thanks 😊
